.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0.0
+
+node-image: &node-image
+  image: cimg/node:14.16.0
+  auth:
+    username: $DOCKERHUB_USERNAME
+    password: $DOCKERHUB_ACCESS_TOKEN
+
+workflows:
+  version: 2
+  workflow:
+    jobs:
+      - build:
+          context:
+            - dockerhub
+            - npm
+          filters:
+            branches:
+              only:
+                - homebound-patch-publish
```

Comment:
This filter allows us to manually trigger the publish whenever we want. We could also set this up to deploy anytime there's a push to the `improved-business-days` branch, but I thought that might get a bit noisy. Thoughts?